### PR TITLE
Mandos storage camelCase keys

### PR DESCRIPTION
**Pull request by andrei-marinica**

### CODE REVIEW

File: contracts/examples/crowdfunding-erc20/mandos/set_accounts.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
                         "''balance|address:erc20_owner": "1,000,000,000",
                         "''balance|address:acc1": "600,000",
                         "''allowance|address:erc20_owner|address:acc1": "400,000",
-                        "''total_supply": "1,000,000,000"
+                        "''totalSupply": "1,000,000,000"
```
Comment (dorin-iancu) [inline contracts/examples/crowdfunding-erc20/mandos/set_accounts.json:34]: str:

File: contracts/examples/crowdfunding-erc20/mandos/set_accounts.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
                     "storage": {
                         "''target": "1,000,000",
                         "''deadline": "123,456",
-                        "''erc20_contract_address": "sc:erc20"
+                        "''erc20ContractAddress": "sc:erc20"
```
Comment (dorin-iancu) [inline contracts/examples/crowdfunding-erc20/mandos/set_accounts.json:44]: str:

**Review by claudiu725 — APPROVED** [review]

File: contracts/examples/ping-pong-egld/mandos/ping-pong-call-pong-all.steps.json [hunk 3]
```diff
@@ -44,15 +44,15 @@
...
+                        "''activationTimestamp": "780",
                         "''deadline": "123,780",
                         "''user_address_to_id|address:participant1": "1",
                         "''user_id_to_address|0x0000001": "address:participant1",
-                        "''user_status|0x0000001": "2",
+                        "''userStatus|0x0000001": "2",
                         "''user_address_to_id|address:participant2": "2",
                         "''user_id_to_address|0x0000002": "address:participant2",
-                        "''user_status|0x0000002": "2",
+                        "''userStatus|0x0000002": "2",
```
Comment (dorin-iancu) [inline contracts/examples/ping-pong-egld/mandos/ping-pong-call-pong-all.steps.json:55]: str:

File: contracts/examples/crowdfunding-erc20/mandos/set_accounts.json (hunk 1 above)
Comment (andrei-marinica) [reply contracts/examples/crowdfunding-erc20/mandos/set_accounts.json:34]: next PR

**Pull request merged**